addons/mail/static/src/new/public/welcome_page.xml

```diff
@@ -9,6 +9,25 @@
         </h1>
         <h2 class="m-5" t-esc="store.companyName"/>
         <div class="d-flex justify-content-center">
+            <!-- <div t-if="props.data.channelData.defaultDisplayMode === 'video_full_screen'" class="position-relative d-flex justify-content-center me-5" t-ref="root"> -->
```

Comment:
comment should be removed?

Reply:
https://github.com/odoo-dev/odoo/pull/2301

Reply:
🙇